RestaurantReservationSystem.Domain/DTOs/Requests/OrderItemRequest.cs

```diff
@@ -1,9 +1,23 @@
 ﻿namespace RestaurantReservationSystem.API.DTOs.Requests
 {
+    /// <summary>
+    /// Represents the data required to create or update an item in an order.
+    /// </summary>
     public class OrderItemRequest
```

Comment:
a general note for later, we would be specific with our requests/responses and just have the properties we need for the API to pass/return, and be specific with the model name i.e. `CreateOrderItemRequest`, `CreateOrderItemResponse`.

Reply:
yeah, but you don't always need a request/response models for each api, we can return other objects directly, but if we you need a request/response model, we need to make it specific to a single operation like `CreateOrderItemRequest`, `CreateOrderItemResponse` with only the needed properties.